package.json

```diff
@@ -18,11 +18,16 @@
     "*.js",
     "test"
   ],
+  "nyc": {
+    "require": ["babel-register"],
+    "reporter": ["text", "html"]
+  },
   "scripts": {
-    "test": "istanbul cover --report cobertura _mocha -- --compilers js:babel-register",
-    "pretest": "godaddy-js-style-lint *.js ./test",
-    "prepublish": "npm run build",
-    "build": "babel *.js -d lib"
+    "build": "babel *.js -d lib",
+    "lint": "eslint --fix *.js test",
```

Comment:
Personally, I find it nice to specify the `.eslint` config at the project root just to make it clear what config this project is using for tools that run `eslint` separately (for example, when Neovim runs it using `neomake`). Of course that's a bit of a selfish reason so I'm happy to remove it.